README.md

```diff
@@ -56,10 +61,49 @@ op.attempt(next => {
 
 ```
 
+### `Async/await` Support
+
+Retryme now supports `async/await` by providing `.async` function to retry async attempt functions.
+
+A simple example of how to use it is as follows:
+
+```js
+
+const Retryme = require('retryme');
+
+const op = new Retryme({
+  retries: 2,
+  min: 50,
+  max: 10000
+}, some_ignore_function_if_exists);
+
+// example of calling the method for async attempt functions
+async function top-caller() {
+  await op.async(async () => {
```

Comment:
this could be a little more tangible and since you aren't using await before returning the `new Promise`. This could be something like:
```js
const readFileAsync = require('util').promisify(require('fs').readFile);
const Retryme = require('retryme');

async function main() {
  const op = new Retryme({
    retries: 2,
    min: 50,
    max: 10000
  }, some_ignore_function_if_exists);

  // Wrap a simple file read with retries.
  try {
    const fileContent = await op.async(() =>readFileAsync(__dirname + '/index.js'))
  } catch(err) {
    console.log('error after 3 attempts');
    console.error(err);
  }
}
```


 I would also give the async function example.
```js
async function foo(bad) {
  if (!bad) return 'bar';
  throw Error('what happened');
}

async function main() {
  const op = new Retryme();
  try {
    const result = await op.async(() => foo());
  } catch(err) {
    console.error(err);
  }
  
}

```

Reply:
with an async function we don't need to invoke it, could just reduce it to `await op.async(foo)`